### PR TITLE
[Intl] Add support for emoji 15.1

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       |
| License       | MIT

* https://blog.emojipedia.org/new-emojis-in-2023/
* https://unicode.org/versions/Unicode15.1.0/
* 4 emojis cannot be build, they have been skipped
     ```
     Could not create transliterator for "🙂‍↔️" in "strip" locale. Code Point: "1f642 200d 2194 fe0f". Error: "transliterator_create_from_rules: unable to create ICU transliterator from rules (parse error at offset 0, before or at "\🙂‍↔️ > test ;"): U_UNQUOTED_SPECIAL".
     Could not create transliterator for "🙂‍↔️" in "strip" locale. Code Point: "1f642 2194 fe0f". Error: "transliterator_create_from_rules: unable to create ICU transliterator from rules (parse error at offset 0, before or at "\🙂‍↔️ > test ;"): U_UNQUOTED_SPECIAL".
     Could not create transliterator for "🙂‍↔" in "strip" locale. Code Point: "1f642 200d 2194". Error: "transliterator_create_from_rules: unable to create ICU transliterator from rules (parse error at offset 0, before or at "\🙂‍↔ > test ;"): U_UNQUOTED_SPECIAL".
     Could not create transliterator for "🙂‍↔" in "strip" locale. Code Point: "1f642 2194". Error: "transliterator_create_from_rules: unable to create ICU transliterator from rules (parse error at offset 0, before or at "\🙂‍↔ > test ;"): U_UNQUOTED_SPECIAL".
     ```
    reproducer:
    ```php
    <?php
    
    var_dump(Transliterator::createFromRules("\\🤐 > test ;"));
    var_dump(Transliterator::createFromRules("\\🙂‍↔️ > test ;")); // This one is broken, don't know why! PHP Bug?
    var_dump(Transliterator::createFromRules("\\🏃🏿‍♀‍➡ > test ;"));
    ```

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/Intl/Resources/emoji/composer.json [hunk 1]
```diff
@@ -14,10 +14,9 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "symfony/filesystem": "^6.4|^7.0",
-        "symfony/finder": "^6.4|^7.0",
-        "symfony/var-exporter": "^6.4|^7.0",
+        "symfony/filesystem": "^6.3|^7.0",
+        "symfony/finder": "^6.3|^7.0",
+        "symfony/var-exporter": "^6.3|^7.0",
```
Comment (fabpot) [inline src/Symfony/Component/Intl/Resources/emoji/composer.json]: Why adding 6.3 here?

Comment (lyrixx) [reply src/Symfony/Component/Intl/Resources/emoji/composer.json]: I started to work on this feature a long time ago! Sorry. 
Fixed

Comment (lyrixx) [reply src/Symfony/Component/Intl/Resources/emoji/composer.json]: Or it was because SF 6.4 was not released

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**